cdk/README.md

```diff
@@ -0,0 +1,47 @@
+# Kafka Connector for Amazon EventBridge Deployment on AWS
+
+This CDK stack is sample deployment to for the Kafka Connector for Amazon EventBridge. 
+
+> :warning: **This sample is not meant to be deployed to production environments!**
+
+## Overview
+
+This sample deploys several components in your AWS account:
+
+- Amazon MSK Serverless Cluster
+- Amazon S3 Bucket
+- IAM Roles & Security Groups
+- VPC & related components
+- Custom Amazon EventBridge Bus
+- Amazon ECS cluster with 2 Services (Data Producer & Transaction Analyzer)
+- (Optional) Amazon MSK Connector
+
+To learn more about those components please read the blog on the [AWS Big Data Blog](https://aws.amazon.com).
+
+![Architecture Diagram](./images/architecture.png)
```

Comment:
nit: on the architecture diagram, the arrow from EventBridge should be inverse, i.e., pointing to the targets because of the push-based invocation (no "consume")